File: PHP/PayPal.php
<?php

require_once('Payment.php');

class Paypal extends Payment
{
    // Atributes

    private $email;

    // Construct

    public function __construct($email)
    {
        parent::__construct();
        $this->email = $email;
    }

    // Getters & Setters 

    public function getEmail()
    {
        return $this->email;
    }

    public function setEmail($email)
    {
        $this->email = $email;
    }

    // Methods
    
}

?>

File: Java/PayPal.java
package Java;

public class PayPal extends Payment
{
    // Atribute

    private String email;

    // Constructor

    public PayPal(String email)
    {
        this.email = email;
    }

    // Getters & Setters

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    // Methods
}